modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/components/RuleBuilder/RulesSupport.es.js

```diff
@@ -59,6 +59,19 @@ const clearAllConditionFieldValues = (condition) => {
 	return condition;
 };
 
+const getFieldOptions = (fieldName, pages) => {
+	let options = [];
+	const visitor = new PagesVisitor(pages);
+
+	const field = visitor.findField((field) => {
+		return field.fieldName === fieldName;
+	});
+
+	options = field ? field.options : [];
+
+	return options;
+}
+
```

Comment:
@alinedoleron, why was it necessary?

Reply:
This method is used both on RulesSupport (inside formatRules method) and RuleEditor (inside prepareStateForRender method).